0.1/index.html

```diff
@@ -1,855 +1,14 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
Is the lowercasing of the `DOCTYPE` intentional? It seems to be valid HTML, but some related tech seems to expect uppercase (see https://stackoverflow.com/a/23946325).